exporters/otlp/otlp.go

```diff
@@ -231,3 +239,51 @@ func (e *Exporter) uploadTraces(ctx context.Context, sdl []*tracesdk.SpanData) e
 	}
 	return err
 }
+
+func NewExportPipeline(ctx context.Context, exportOpts []ExporterOption,
+	resOpts []resource.Option) (*Exporter, *sdktrace.TracerProvider, *push.Controller, error) {
+
+	exp, err := NewExporter(ctx, exportOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	res, err := resource.New(ctx, resOpts...)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	bsp := sdktrace.NewBatchSpanProcessor(exp)
+	tracerProvider := sdktrace.NewTracerProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithResource(res),
+		sdktrace.WithSpanProcessor(bsp),
+	)
+
+	pusher := push.New(
+		basic.New(
+			simple.NewWithExactDistribution(),
+			exp,
+		),
+		exp,
+		push.WithPeriod(7*time.Second),
```

Comment:
The default is 10 seconds: https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/metric/controller/push/push.go#L31 

Can we just use that instead of overriding it?

Reply:
sure, i will update to default